src/styles/index.scss

```diff
@@ -1,3 +1,72 @@
-body {
+@import 'variables';
+@import 'mixin';
+
+@keyframes move {
+  from {
+    transform: rotate(0deg);
+  }
+
+  to {
+    transform: rotate(360deg);
+  }
+}
+
+.page {
   margin: 0;
+  height: 100%;
+
+  @include center_the_element;
+}
+
+.stopwatch {
+  width: $stopwatch_size;
+  height: $stopwatch_size;
+  border: 1vmin dotted #000;
+  border-radius: 50%;
+
+  @include center_the_element;
+}
+
+.stopwatch_center {
+  z-index: 1;
+  width: $center_size;
+  height: $center_size;
+  background-color: #f6a603;
+  border-radius: 50%;
+}
+
+.hand {
+  position: absolute;
+
+  &_minutes {
+    position: absolute;
+    transform-origin: bottom;
+    height: 20vmin;
+    width: 3vmin;
+    background-color: #0700ff;
+    top: -20vmin;
+    left: -1.5vmin;
+
+    animation: move 3600s steps(60) infinite;
```

Comment:
The `&` character is used to concatenate the parent selector, but it should be followed by a modifier or an element according to BEM. In this case, `_minutes` seems to be an element, but it should be `__minutes` to follow the BEM naming convention. The same applies to `_seconds` on line 53.